packages/components/organisms/f-cookie-banner/test/component/f-cookie-banner-legacy.component.spec.js

```diff
@@ -8,41 +8,43 @@ describe('Legacy - f-cookie-banner component tests', () => {
     beforeEach(async () => {
         // Arrange
         cookieBanner = new CookieBanner();
-        await cookieBanner.withQuery('args', 'locale:en-AU');
+    });
 
+    it('should display the f-cookie-banner legacy component', async () => {
         // Act
-        await cookieBanner.load();
-    });
+        cookieBanner.open('/');
+        await browser.deleteCookies();
+        await cookieBanner.load({ locale: 'en-AU', shouldShowLegacyBanner: true });
 
-    it('should display the f-cookie-banner component', async () => {
         // Assert
-        await expect(await cookieBanner.isCookieBannerComponentDisplayed()).toBe(true);
+        await expect(await cookieBanner.isComponentDisplayed()).toBe(true);
     });
 
     it('should set "je-cookie_banner" cookie when dismissed.', async () => {
         // Act
+        cookieBanner.open('/');
+        await browser.deleteCookies();
+        await cookieBanner.load({ locale: 'en-AU', shouldShowLegacyBanner: true });
         await cookieBanner.close();
 
         // Assert
         const [bannerCookie] = (await browser.getCookies()).filter(cookie => cookie.name === 'je-banner_cookie');
         await expect(bannerCookie.value).toBe('130315');
-        await expect(await cookieBanner.isCookieBannerComponentDisplayed()).toBe(false);
+        await expect(await cookieBanner.isComponentDisplayed()).toBe(false);
     });
 
     forEach([
         ['en-AU', 'au/info/privacy-policy#cookies_policy'],
         ['en-NZ', 'nz/info/privacy-policy#cookies_policy']
     ])
-    .it('should go to the correct cookie policy page for "%s" - "%s"', async (tenant, expectedCookiePolicyUrl) => {
-        // Arrange
-        cookieBanner = new CookieBanner();
-        await cookieBanner.withQuery('args', `locale:${tenant}`);
-
+    .it('should go to the correct cookie policy page for "%s" - "%s"', (tenant, expectedCookiePolicyUrl) => {
```

Comment:
minor - think some `awaits` might be missing in this test

Reply:
Await causes inconsistent failures, I presume due to it creating race conditions.